src/core/logical_layers/renderers/BivariateRenderer.tsx

```diff
@@ -47,23 +47,26 @@ type FillColor = {
 
 const MAX_BIVARIATE_GENERATED_RESOLUTION = 8; // keep in sync with backend
 
-const HOVER_HEXAGON_BORDER = 'rgba(5, 22, 38, 0.4)';
-const ACTIVE_HEXAGON_BORDER = 'rgb(5, 22, 38)';
+export const HOVER_HEXAGON_BORDER = 'rgba(5, 22, 38, 0.4)';
+export const ACTIVE_HEXAGON_BORDER = 'rgb(5, 22, 38)';
 
-const HEX_SELECTED_LAYER_ID = 'hex-selected-layer';
-const HEX_SELECTED_SOURCE_ID = 'hex-selected-source';
-const HEX_HOVER_SOURCE_ID = 'hex-hover-source';
-const HEX_HOVER_LAYER_ID = 'hex-hover-layer';
+export const HEX_SELECTED_LAYER_ID = 'hex-selected-layer';
+export const HEX_SELECTED_SOURCE_ID = 'hex-selected-source';
+export const HEX_HOVER_SOURCE_ID = 'hex-hover-source';
+export const HEX_HOVER_LAYER_ID = 'hex-hover-layer';
```

Comment:
Maye it's good time to move it to constants.ts

Reply:
this is scheduled for removal when redoinf the highlights using MVT features

Reply:
yeah, after features will have an ids we'll be able to highlight them without any additional layers
https://docs.mapbox.com/mapbox-gl-js/example/hover-styles/